src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import { Progress } from '../../types/Progress';
+
+interface Props {
+  itemsLeft: number,
+  setProgress: (type: Progress) => void;
+}
+
+export const Footer:React.FC<Props> = (props) => {
+  const { itemsLeft, setProgress } = props;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${itemsLeft} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
```

Comment:
Do not leave unnecessary comments.
Fix this in all code.

Reply:
not fixed